app/extensions.js

```diff
@@ -363,6 +364,25 @@ module.exports.init = () => {
     extensionActions.extensionEnabled(installInfo.id)
   })
 
+  let insertLocaleStrings = (installInfo) => {
+    let pattern = /^__MSG_(.*)__$/
+    let properties = ['name', 'description']
+    let defaultLocale = installInfo.manifest.default_locale
+    if (defaultLocale) {
+      let msgPath = path.join(installInfo.base_path, '_locales', defaultLocale, 'messages.json')
+      if (fs.existsSync(msgPath)) {
+        let messages = JSON.parse(fs.readFileSync(msgPath).toString())
+        properties.forEach((property) => {
+          let key = installInfo[property].match(pattern)[1]
```

Comment:
@cezaraugusto What do you mean by _toggling_? Were you enabling and disabling the extension(s) in Preferences?